vkcs/networking/data_source_sdn.go

```diff
@@ -0,0 +1,45 @@
+package networking
+
+import (
+	"context"
+
+	"github.com/gophercloud/utils/terraform/hashcode"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/clients"
+	isdn "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/networking"
+)
+
+func DataSourceNetworkingSDN() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceNetworkingSDNRead,
+
+		Schema: map[string]*schema.Schema{
+			"sdn": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "Names of available VKCS SDNs in the current project.",
+			},
+		},
+		Description: "Use this data source to get a list of available VKCS SDNs in the current project.",
```

Comment:
lets add
The first SDN is default. You do not have to specify default sdn argument in resources and datasources. You may specify non default SDN only for root resources such as `vkcs_networking_router`, `vkcs_networking_network`, `vkcs_networking_secgroup` (they do not depend on any other resource/datasource with sdn argument).

Reply:
add